tests/suites/test_suite_asn1write.data

```diff
@@ -67,14 +67,26 @@ mbedtls_asn1_write_len:255:"81FF":2:2
 ASN.1 Write / Read Length #5 (Len = 255, buffer too small)
 mbedtls_asn1_write_len:255:"81FF":1:MBEDTLS_ERR_ASN1_BUF_TOO_SMALL
 
-ASN.1 Write / Read Length #6 (Len = 256, long form)
-mbedtls_asn1_write_len:256:"820100":3:3
+ASN.1 Write / Read Length #6 (Len = 258, byte order)
```

Comment:
With 0xFFFF you cannot see if the bytes were place in the correct order into the ASN1 representation.  With 0x0102 you can.

-------- Original message --------
From: Janos Follath <notifications@github.com>
Date: 7/15/16 15:48 (GMT+01:00)
To: ARMmbed/mbedtls <mbedtls@noreply.github.com>
Cc: Paul Bakker <Paul.Bakker@arm.com>, Author <author@noreply.github.com>
Subject: Re: [ARMmbed/mbedtls] Coverage improvements (#540)


In tests/suites/test_suite_asn1write.data<https://github.com/ARMmbed/mbedtls/pull/540#discussion_r70975243>:

> @@ -67,14 +67,26 @@ mbedtls_asn1_write_len:255:"81FF":2:2
>  ASN.1 Write / Read Length #5 (Len = 255, buffer too small)
>  mbedtls_asn1_write_len:255:"81FF":1:MBEDTLS_ERR_ASN1_BUF_TOO_SMALL
>
> -ASN.1 Write / Read Length #6 (Len = 256, long form)
> -mbedtls_asn1_write_len:256:"820100":3:3
> +ASN.1 Write / Read Length #6 (Len = 258, byte order)


What does byte order mean here?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ARMmbed/mbedtls/pull/540/files/7d1ad315a8570948c2b60e9b59e764e3036d0fa8..9d63b7774ba82ad9d1721c9cb7fca8e5e719f87c#r70975243>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABNYRIVQ0RrrO7gTOFq08lYxJgK1co8Zks5qV4-AgaJpZM4JMg88>.

IMPORTANT NOTICE: The contents of this email and any attachments are confidential and may also be privileged. If you are not the intended recipient, please notify the sender immediately and do not disclose the contents to any other person, use it for any purpose, or store or copy the information in any medium. Thank you.